src/components/Listings/Listings.scss

```diff
@@ -0,0 +1,12 @@
+.listing-section {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.listing-card {
+  border: .5em solid black;
+  width: 30%;
```

Comment:
Stay away from `%` measurements for stuff like this. There are easier ways to approximate equal sizing for your layout using grid or flexbox. You're doing a good job with using `em` and the like, so stay consistent.

If you need a static width, use `em` and then accommodate other devices/responsivity with breakpoints. If you set it up right with grid/flexbox, your responsiveness will be 🍰

Reply:
This is definitely something I'd like a bit more direction on how to do effectively!